src/garage/replay_buffer/multi_task_replay_buffer.py

```diff
@@ -0,0 +1,145 @@
+"""A replay buffer memory for meta-RL."""
```

Comment:
@krzentner PTAL ASAP

Reply:
I don't fully understand the role of this class in the context of PEARL yet. It seems to be fine to have a replay buffer that keeps each tasks' samples in a separate buffer, but I don't quite get why it's useful. This seems like a reasonable way to implement it.

Reply:
The algorithm keeps a buffer for each task, and I think this looks cleaner. I just followed the implementation on oyster, but I suppose I can keep a list of buffers in the algo without this buffer class.

Reply:
this looks generally fine to me, but how does it work for meta-RL on continuous spaces? i.e. HalfCheetahRandomVel has "infinite" tasks, because you sample a new meta-batch of environments (velocities) on every meta-batch